airflow/utils/email.py

```diff
@@ -21,13 +21,16 @@
 from past.builtins import basestring
 
 import importlib
+import mimetypes
 import os
+import sendgrid
 import smtplib
 
 from email.mime.text import MIMEText
 from email.mime.multipart import MIMEMultipart
 from email.mime.application import MIMEApplication
 from email.utils import formatdate
+from sendgrid.helpers.mail import *
```

Comment:
Importing * is considered bad form in python -- it makes it hard to know what comes from which module.

Reply:
Good catch, updated.